tests/e2e/specs/backend/cart.test.js

```diff
@@ -51,6 +54,7 @@ describe( `${ block.name } Block`, () => {
 			beforeAll( async () => {
 				// make sure CartCheckoutCompatibilityNotice will appear
 				await page.evaluate( () => {
+					// eslint-disable-next-line no-undef
```

Comment:
Instead of this line (and others for the same `no-undef` disables), could we maybe use `window.localStorage` in the eval function? Seems to get rid of the error. Is there a reason why we aren't accessing it through `window` though?

Reply:
Thanks for noticing this. [I created an issue for this](https://github.com/woocommerce/woocommerce-blocks/issues/6442). I will create a dedicated PR this week 👍